docs/content/preview/deploy/multi-dc/async-replication/includes/automatic-setup.md

```diff
@@ -0,0 +1,166 @@
+<!--
++++
+private = true
++++
+-->
+
+Since this feature is in [technical preview](/preview/releases/versioning/#tech-preview-tp), you must enable it by adding the `xcluster_enable_ddl_replication` flag to the [allowed_preview_flags_csv](../../../reference/configuration/yb-master/#allowed-preview-flags-csv) list and setting it to true on yb-master.
```

Comment:
```suggestion
Since this feature is in [technical preview](/preview/releases/versioning/#tech-preview-tp), you must enable it by adding the `xcluster_enable_ddl_replication` flag to the [allowed_preview_flags_csv](../../../reference/configuration/yb-master/#allowed-preview-flags-csv) list and setting it to true on yb-master in both universes.
```